lib/BloqadeSchema/schema_examples/tutorial_examples/1D_scar.jl

```diff
@@ -17,8 +17,13 @@ atoms = generate_sites(ChainLattice(), nsites, scale = 5.72)
 Ω_tot = append(Ω1, Ω2);
 Δ_tot = append(Δ1, Δ2);
 
-H = rydberg_h(atoms; Δ = Δ_tot, Ω = Ω_tot)
-h = to_json(H,waveform_tolerance=1e-1, warn=true)
+ϕ = constant(duration = 4.2, value = 0);
+
+h = rydberg_h(atoms; Δ = Δ_tot, Ω = Ω_tot, ϕ=ϕ)
+device_capabilities = get_device_capabilities()
+device_capabilities.rydberg.global_value.timeMax = 5
```

Comment:
```suggestion
# WARNING: This will result in invalid JSON file for actual QPU. 
device_capabilities.rydberg.global_value.timeMax = 5
```

Just adding a comment for the user about what this does.